Master-OB-OpenAPI.json

```diff
@@ -5381,7 +5395,9 @@
             "description": "A product code is a standardized, unique human-readable identifier that is compact,and can be easily parsed. It consists of an entity code concatenated to a product specific identification string, separated by a hyphen. A product string is up to 16 characters, comprised of upper case letters, number, and hyphens (Ã¢â‚¬Ëœ-Ã¢â‚¬Ëœ).\nThe format of a product code is:\n[EntityCode]-[ProductString], e.g., UEE-E2000-3-XFM for company Universal Energy Enterprises with Entity Code UEE and Product String of E2000-3-XFM.    ",
             "x-ob-item-type": "StringItemType",
             "x-ob-usage-tips": "",
-            "x-ob-sample-value": {},
+            "x-ob-sample-value": {
+              "Value": "IQ7PLUS-72-ACM-US"
```

Comment:
This isn't a valid product code example, per our description. The manufacturer is Enphase. "IQ7PLUS-72-ACM-US" is the Product String and needs to be prefixed by the Entity Code for Enphase.

This illustrates the concern I have about our Product code definition, that OB is requiring an Entity Code that a manufacturer doesn't use and that customers won't know.

Reply:
Update description to include entity code of 6 digits and no limit on product code.

Reply:
Example: SMA-SB7.7-1SP-US-41-NWF